wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1

```diff
@@ -322,3 +322,5 @@ if ($WebhookURI) {
 else {
 	Write-Warning "A Webhook URI has already been generated for this Automation account. Please use the existing Webhook URI to setup the execution schedule for the Azure Logic App. The URI is also stored as encrypted in the above Automation Account variable. To retrieve the value, see https://docs.microsoft.com/en-us/azure/automation/shared-resources/variables?tabs=azure-powershell#powershell-cmdlets-to-access-variables"
 }
+
+return $WebhookURI
```

Comment:
@SvenAelterman This may not work as intended by the caller because there are other things that this function outputs using `Write-Output` which would also get returned to the caller of the this function.

Reply:
You're correct, I realized that after submitting the PR. I'll see about an alternate solution.